components/shared/net/Cargo.toml

```diff
@@ -23,7 +23,7 @@ headers = { workspace = true }
 http = { workspace = true }
 hyper = { workspace = true }
 hyper_serde = { workspace = true }
-hyper-util = { workspace = true, features = ["client-legacy"] }
+hyper-util = { workspace = true, features = ["client-legacy", "http2"] }
```

Comment:
Do you mind moving all of these features to the root `Cargo.toml` as well?

Reply:
Like this?